src/pages/Profile.jsx

```diff
@@ -1,8 +1,11 @@
 import React from "react";
+import { FormattedMessage } from "react-intl";
 
 const Profile = () => (
   <div className="Profile" data-test="Profile-page">
-    <h1>Profile</h1>
+    <h1>
+      <FormattedMessage id="Profile.first-header" defaultMessage="Profile" />
+    </h1>
```

Comment:
🔧 add a test for this

Reply:
This component no longer contains this element:  c0dd94230ccd7f24127e61d1966455951fa0dd42